README.md

```diff
@@ -17,8 +17,8 @@ members of this organization are candidate reviewers.
 * select scopes: `repo` & `public_repo` & `read:org`
 
 # installation
-* install this npm package to your hubot repo
-    * `npm i --save hubot-reviewer-queue`
+* install this package to your hubot repo
+    * copy this repo and paste it into your hubot repo's node_modules folder as `hubot-reviewer-queue`
```

Comment:
This can be in a separate dev section, but the install procedure should remain the same for consumers of this package.

Reply:
Done. Reverted.